pages/board.js

```diff
@@ -1,3 +1,20 @@
-export default function BoardPage() {
-  return <h1>Board</h1>;
+import { useState } from "react";
+import TaskCard from "../components/TaskCard/TaskCard";
+import styled from "styled-components";
+
+export default function Board({ tasks }) {
+  console.log(tasks);
+  return (
+    <>
+      <CardContainer>
+        {tasks.map((task) => (
+          <TaskCard key={task.title} task={task} />
```

Comment:
the task title might not be the best candidate for `key`. Keys have to be unique per list, so if you have two tasks with the same title React will rightfully complain.

Consider adding an `id` to your tasks to have a unique identifier.

To do so install `uid`

```
npm i uid
```

and then change your `handleAddTask` function in `_app.js` to create ids:

```js
import { uid } from "uid";

/* … */

  function handleAddTask(newTask) {
    setTasks([...tasks, {...newTask, id: uid()}]);
  }
```

then you can do:
```suggestion
          <TaskCard key={task.id} task={task} />
```